lib/host/signals/local_storage.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const { createWriteStream } = require('fs')
+const { stat, mkdir } = require('fs/promises')
```

Comment:
@tarcisiozf @avsek477 this doesn't seem to be supported by node v12, see https://github.com/bitfinexcom/bfx-hf-ui-core/runs/5248400666?check_suite_focus=true
please use require('fs').promises to have backward compatibility on future

Reply:
We were planning to upgrade to node 14 and we had to fallback, will fix